infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -6,6 +6,7 @@
 import java.util.HashMap;
 import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)